_posts/2021-12-20-changelog-162.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #162"
+date: 2021-12-20 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8145] Take into account [const generics defaults] in const evaluation
+
+* [#8070] Suggest private items when completing code in the debugger
+
+* [#8192] Parse and annotate [`~const` syntax]
+
+## Fixes
+
+* Procedural macro expansion fixes:
+
+  * [#8271] Fix expansion of `sqlx` procedural macros with `offline` feature
```

Comment:
Do I understand correctly that this fix is not only about `sqlx` library but about any proc macro that writes code using `CARGO_TARGET_DIR` variable?
If it's true, let's write it and make "`sqlx` procedural macros with `offline` feature" as example